src/tlo/methods/epilepsy.py

```diff
@@ -406,8 +411,14 @@ def do_at_generic_first_appt_emergency(
         **kwargs,
     ) -> None:
         if "seizures" in symptoms:
-            event = HSI_Epilepsy_Start_Anti_Epileptic(person_id=person_id, module=self)
-            schedule_hsi_event(event, priority=0, topen=self.sim.date)
+            # Determine if treatment will start - depends on probability of prescribing, which is calibrated to
+            # induce the right proportion of persons with epilepsy receiving treatment.
+
+            prob_start = self.parameters['prob_start_anti_epilep_when_seizures_detected_in_generic_first_appt']
+
+            if self.rng.random_sample() < prob_start:
+                event = HSI_Epilepsy_Start_Anti_Epileptic(person_id=person_id, module=self)
+                schedule_hsi_event(event, priority=0, topen=self.sim.date)
 
```

Comment:
Wondering if, after these changes, cons use is still too high that we could limit the number of times an individual can return to this HSI to start treatment - currently unlimited?

Reply:
It's a good idea, Joe -- I'll add that in.

I don't think this is a major cause of over-demand as the logic selects the 'best available' of the meds, and only reschedules if non-available. Nevertheless, it's a good thing to do.